source/web-hosting/applications.rst

```diff
@@ -81,6 +81,31 @@ To install dependencies::
 
 Then see ``nodejs/app.js`` for a minimum base application.
 
+Forwarding requests to your application server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The SRCF uses Apache to serve websites so if you need to forward requests to a backend web app, for example a Django, Rails or Express server, then you will need to forward web requests. Make sure you run your application server on ``sinkhole`` rather than ``pip`` or another SRCF server.
```

Comment:
The first sentence is a bit tautological - "if you need to forward requests you need to forward requests".

Probably replace "forward requests to a backend" with "run a backend"?